planning/behavior_path_planner/src/utils/path_shifter/path_shifter.cpp

```diff
@@ -555,17 +555,29 @@ double PathShifter::calcShiftTimeFromJerk(const double lateral, const double jer
 }
 
 double PathShifter::calcFeasibleVelocityFromJerk(
-  const double lateral, const double jerk, const double distance)
+  const double lateral, const double jerk, const double longitudinal)
```

Comment:
not related this PR but suggetion for variable name also for the function of calcLateralDistFromJerk
```suggestion
double PathShifter::calcFeasibleVelocityFromJerk(
  const double lateral, const double jerk, const double longitudinal)
```
```
double PathShifter::calcFeasibleVelocityFromJerk(
  const double lateral_deviation const double longitudinal_deviation, const double maximum_jerk)
```